src/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ configure_file(csympy_config.h.in csympy_config.h)
 include_directories(BEFORE ${csympy_BINARY_DIR}/src)
 
 add_library(csympy ${SRC})
+target_link_libraries(csympy teuchos ${LIBS})
```

Comment:
a) if `csympy` is a static lib, I think this changes nothing.
b) what you said is correct